README.md

```diff
@@ -23,7 +23,19 @@ In your terminal, go to the project directory
 - Install Bundler if this gem is not installed yet :
 
 ```
-gem install Bundler
+gem install bundler
+```
+
+- Install cocoapods if this gem is not installed yet :
+
+```
+gem install cocoapods
```

Comment:
This is not needed.. 
Cocoapods can be found in the Gemfile and will be installed by the command 'bundle install --path vendor/bundle'